mmv1/third_party/terraform/services/secretmanager/resource_secret_manager_secret_version_test.go

```diff
@@ -40,7 +40,7 @@ func TestAccSecretManagerSecretVersion_update(t *testing.T) {
 				ImportStateVerifyIgnore: []string{"secret_data", "secret_data_wo_version"},
 			},
 			{
-				Config: testAccSecretManagerSecretVersion_basic(context),
+				Config: testAccSecretManagerSecretVersion_byName(context),
```

Comment:
Can this be it's own test, unfortunately since we suppress the diff on the secret it's hiding the functionality here on creation.